### PR TITLE
Add NEXX to Junotestnet-Osmosistestnet.json

### PR DESCRIPTION
**Pull request by NEXXGENAI**

### CODE REVIEW

File: testnets/_IBC/junotestnet-osmosistestnet.json [hunk 1]
```diff
@@ -28,6 +28,38 @@
         "dex": "osmosis"
       }
     }
-  ]
-}
-  
\ No newline at end of file
+  ],
```
Comment (JeremyParish69) [inline testnets/_IBC/junotestnet-osmosistestnet.json:31]: this doesn't follow the schema, as it only supports one set of connection ids and client ids. Please use the same client+connection ids, and set up the channel using those.

File: testnets/_IBC/junotestnet-osmosistestnet.json [hunk 2]
```diff
@@ -28,6 +28,38 @@
...
+      "chain_2": {
+        "chain_name": "osmosistestnet",
+        "client_id": "07-tendermint-832",
+        "connection_id": "connection-739"
+      },
+      "channels": [
+        {
+          "chain_1": {
+            "channel_id": "channel-622",
+            "port_id": "transfer"
```
Comment (JeremyParish69) [inline testnets/_IBC/junotestnet-osmosistestnet.json:47]: I'm surprised to see the transfer port id, when I'm used to seeing that cw20s need a contract and wasm.contract port for moving cw20s

File: testnets/_IBC/junotestnet-osmosistestnet.json (hunk 1 above)
Comment (piscesyuma) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: Hi JeremyParish69, 
We've created a new relayer between juno testnet(uni-6) and osmosis testnet(osmo-test-5).
how can we add our own new set of connection ids and client ids?
We can not find and use the connection id and client id that was already registered at junotestnet-osmosistestnet.json file.

File: testnets/_IBC/junotestnet-osmosistestnet.json (hunk 2 above)
Comment (piscesyuma) [reply testnets/_IBC/junotestnet-osmosistestnet.json:47]: I think juno factory token is not follow cw20 format.

File: testnets/_IBC/junotestnet-osmosistestnet.json (hunk 1 above)
Comment (piscesyuma) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: The relayer info(channel id, client id, connection id) already registered does not exists.
Is it possible for us to swap original relayer info to our new relayer info for junotestnet-osmosistestnet.json?

File: testnets/_IBC/junotestnet-osmosistestnet.json (hunk 2 above)
Comment (JeremyParish69) [reply testnets/_IBC/junotestnet-osmosistestnet.json:47]: That's correct, my mistake

File: testnets/_IBC/junotestnet-osmosistestnet.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: I'll need to investigate further to prove it, but I believe it does indeed exist...

Comment (piscesyuma) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: I've already investigated about the existence of already registered channel.
If it was exists, we didn't need to make a new relayer between uni6 to osmo-test-5.

image

Comment (NEXXGENAI) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: > I'll need to investigate further to prove it, but I believe it does indeed exist...

It does not appear to exist, which is why we created a new relayer between uni-6 and osmo-test5.  Please provide further direction so we can get this moving.  Thank you

Comment (JeremyParish69) [reply testnets/_IBC/junotestnet-osmosistestnet.json:31]: The original client maybe be revived via governance

**Pull request merged**